src/models/galp7/internal-overview.md

```diff
@@ -0,0 +1,26 @@
+# Galago Pro (Internal Overview)
+
+## Bottom components and connectors:
+
+![Internal Components](./img/components-highlighted.webp)
+
+- RAM slots are highlighted in purple
+- M.2 SSD is highlighted in maroon
+- Wireless card is highlighted in yellow
+- CMOS battery connector is highlighted in pink
+- Webcam/microphone connector is highlighted in cyan
+- BIOS flash chip (U43) is highlighted in black
+- Fan connectors are highlighted in red
+- Speaker connectors are highlighted in dark green
```

Comment:
Is only one connection accessible?

Reply:
I don't know what you mean. There are two connection shown.

The one for the right speaker (on the left) is a little farther away, on the opposite side of the wireless card.